src/regex/pcre2.cr

```diff
@@ -279,12 +276,6 @@ module Regex::PCRE2
     match_data
   end
 
-  def self.config(what, type : T.class) : T forall T
-    value = uninitialized T
-    LibPCRE2.config(what, pointerof(value))
-    value
-  end
-
   module MatchData
```

Comment:
What's the reason for this removal? It isn't used? Is there a relation with #13307?

Reply:
It's unused, `LibPCRE2.config` is directly called